### DISCUSSION TITLE
Galaxy Brain Achievement

### DISCUSSION THREAD

**Discussion by jokefrelon** [Schweinepriester/github-profile-achievements#312] [Q&A] [answered]
i want the Galaxy Brain Achievement!🥺

**Comment by scarabaeus** [comment, marked as answer]
Let's do it! @jokefrelon  You can answer this one https://github.com/Schweinepriester/github-profile-achievements/discussions/313

**Reply by jokefrelon** [reply]
okay

**Comment by WHAHA-HA** [comment]
Actually, I can achieve Quickdraw and Pull Shark badges with ease, but no Galaxy Brain.
If you want me to do anything, feel free to let me know anytime
PS: It'd be appreciated if you could mark my answer
